registration-form/src/App.jsx

```diff
@@ -1,16 +1,34 @@
-import { useState } from 'react'
-import './App.css'
+import style from "./App.module.css";
+import { useState } from "react";
+import { FormMain } from "./components/FormMain/FormMain";
+import { FormResume } from "./components/FormResume/FormResume";
 
 function App() {
+	const [isFormSubmitted, setIsFormSubmitted] = useState(false);
```

Comment:
In my opinion, this form is unnecessary because this information is stored in the formData state.